app/main.js

```diff
@@ -27,7 +27,9 @@ const {version: localVersion} = require(app.getAppPath() + '/package.json');
 
 const VERSION_CHECK_INTERVAL = 30 * 60 * 1000;
 const LATEST_RELEASE_API_URL = 'https://api.github.com/repos/lbryio/lbry-app/releases/latest';
+const DAEMON_PATH = process.env.LBRY_DAEMON || path.join(__dirname, 'dist', 'lbrynet-daemon');
 
+global.daemonPath = DAEMON_PATH; // For easy access from render process
```

Comment:
I'm going to scratch all the daemon-related code completely. I just didn't want to clean it all out until I was sure we're nixing the original approach of calling the daemon.